server.go

```diff
@@ -820,11 +820,12 @@ func newServer(listenAddrs []net.Addr, chanDB *channeldb.DB,
 	})
 
 	// Construct a closure that wraps the htlcswitch's CloseLink method.
+	// Provide no close out delivery address, as user input is not available.
 	closeLink := func(chanPoint *wire.OutPoint,
 		closureType htlcswitch.ChannelCloseType) {
 		// TODO(conner): Properly respect the update and error channels
 		// returned by CloseLink.
-		s.htlcSwitch.CloseLink(chanPoint, closureType, 0)
+		s.htlcSwitch.CloseLink(chanPoint, closureType, 0, nil)
```

Comment:
readability nit: consider adding a variable eg. `deliveryAddress := nil` and the top comment above that, so that it's even more clear what `nil` refers to.

Reply:
Declaring a var just so we can pass in a named nil variable is a bit atypical for go, going to leave this one if that's ok? Rather going to move the comment down so it's closer to the nil value.

Reply:
Agreed that it is a bit hard to read just by passing nil, but a comment seem sufficient  in this case :)